xtb_6.5.0 cannot delegate work to ORCA

Dear all,

it seems that after the upgrade from 6.4.1 to 6.5.0 xtb it is no more capable to delegate portion of work to ORCA 5.0.3 (nor 5.0.2, nor 5.0.1), though ORCA itself runs well when not called from within of xtb.

I tested also switching between CENSO 1.1.2 and 1.2.0 where a similar problem occurred earlier and has been resolved (https://github.com/grimme-lab/CENSO/issues/19)

Relevant files are attached here:
[censo120-xtb641.zip](https://github.com/grimme-lab/xtb/files/9028797/censo120-xtb641.zip)
[censo120-xtb650.zip](https://github.com/grimme-lab/xtb/files/9028800/censo120-xtb650.zip)
[censo112-xtb641.zip](https://github.com/grimme-lab/xtb/files/9028802/censo112-xtb641.zip)
[censo112-xtb650.zip](https://github.com/grimme-lab/xtb/files/9028803/censo112-xtb650.zip)

Thanks for any advice

Looks like the sanity check Albert @Albkat and me implemented is causing problems here

```
########################################################################
[ERROR] Program stopped due to fatal error
-2- Could not setup single-point calculator
-1- Executable '/usr/local/orca503/orca' is in the system path, is this the GNOME screen reader?
########################################################################
```

We checker whether the orca binary is in a subdirectory of `/usr`, which is very likely to be [a different orca](https://help.gnome.org/users/orca/stable/index.html.en) (the GNOME screenreader) and therefore we abort the run.

Installing Orca in any other prefix (`/opt` or `~/.local`) should solve this issue.

cc @MtoLStoN

--- a/src/extern/orca.f90
+++ b/src/extern/orca.f90
@@ -95,9 +95,12 @@ subroutine checkOrca(env, ext)
    endif
 
    if (index(ext%executable,'/usr') == 1) then
-      call env%error("Executable '"//ext%executable//"' is in the system path, "//&
-         & "is this the GNOME screen reader?")
-      return
+      inquire(file=ext%executable//'_scf',exist=exist)
+      if (.not. exist) then
+         call env%error("Executable '"//ext%executable//"' is in the system path, "//&
+            & "is this the GNOME screen reader?")
+         return
+      endif
    endif
 
    ! see if there is a preference for an input file

